source/features/pr-branches.tsx

```diff
@@ -82,35 +86,35 @@ function buildQuery(numbers: number[]) {
 function createLink(ref: RepositoryReference) {
 	return (
 		<span
-			class="commit-ref css-truncate user-select-contain mb-n1"
-			style={(ref.branchExists ? {} : {'text-decoration': 'line-through'})}>
+			className="commit-ref css-truncate user-select-contain mb-n1"
+			style={(ref.branchExists ? {} : {textDecoration: 'line-through'})}>
```

Comment:
The react-types were complaining about this, so I converted it to camelCase. Does dom-chef handle this @bfred-it? If not will have to take this into consideration for typing the `JSX` types properly for `dom-chef`.

Reply:
This is ok, `el.style` supports both ways.